rate_counter/rate/views.py

```diff
@@ -1,3 +1,161 @@
-from django.shortcuts import render
+from django.shortcuts import render, HttpResponseRedirect, get_object_or_404, get_list_or_404
+from .models import Institute, Subject, StudentGroup, Table, TableRow
+from .forms import TableForm
 
 # Create your views here.
+
+
+def create_table(request):
+    """
+    creation of new table
+    """
+    form = TableForm(request.POST or None)
+    if form.is_valid():
+        instance = form.save(commit=False)
+        instance.save()
+        return HttpResponseRedirect("/rate/detail/%s/" % instance.id)
+    context = {
+        'form': form
+    }
+    return render(request=request, template_name="table_form.html", context=context)
+
+
+def edit_table(request, id=None):
+    """
+    edit of existing table
+    """
+    table_obj = get_object_or_404(Table, id=id)
+    form = TableForm(request.POST or None, request.FILES or None, instance=table_obj)
+    if form.is_valid():
+        instance = form.save(commit=False)
+        instance.save()
+        return HttpResponseRedirect("/rate/detail/%s/" % instance.id)
```

Comment:
If you will use 'named' urls it will make this more cleaner. [Example](https://docs.djangoproject.com/en/dev/ref/urlresolvers/#django.urls.reverse) with reversing